package.json

```diff
@@ -23,31 +23,24 @@
   "bugs": {
     "url": "https://github.com/kuzzleio/kuzzle-plugin-auth-github/issues"
   },
-  "pluginInfo": {
-    "defaultConfig": {
-      "strategies": {}
-    }
-  },
   "license": "Apache-2.0",
   "dependencies": {
     "passport": "0.3.2",
     "passport-oauth2": "1.x.x",
     "passport-oauth2-refresh": "1.0.0",
-    "q": "2.0.3"
+    "q": "2.0.3",
```

Comment:
// TODO: replace `q` with `bluebird`

I know this is not exactly the right PR to do that, but do you think you can apply this change?
`q` is really slow and memory consuming compared to other promise modules.